test/dom-builder.spec.ts

```diff
@@ -35,6 +41,29 @@ describe("dom-builder", () => {
     expect(domBtn).to.have.html("<button></button>");
   });
 
+  it("basic SVG elements", () => {
```

Comment:
This is nitpicking and I don't care much about it, but if there's an easy way to do it making the "it" and the string read like a sentence is a nice. Something like "it renders basic SVG elements" for instance.